locations/spiders/starbucks_ae.py

```
@@ -9,16 +11,10 @@ class StarbucksAESpider(SitemapSpider, StructuredDataSpider):
     name = "starbucks_ae"
     item_attributes = {"brand": "ستاربكس", "brand_wikidata": "Q37158"}
     sitemap_urls = ["https://locations.starbucks.ae/robots.txt"]
-    sitemap_rules = [(r"ae/directory/[^/]+/[^/]+$", "parse")]
+    sitemap_rules = [(r"starbucks\.ae/(?!ar/)[^/]+/[^/]+$", "parse")]
```

Ruff (0.16.1)

[warning] Mutable default value for class attribute

(RUF012)